sale_order_product_assortment/models/sale_order_line.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2020 Tecnativa - Carlos Roca
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+from odoo import api, fields, models
+
+
+# We overwrite this class as a trick to repeat the product_id field with different
+# attributes on the view.
+class SaleOrderLine(models.Model):
+    _inherit = 'sale.order.line'
+
+    product_2_id = fields.Many2one(
```

Comment:
Rename the fields to represent what they mean.

- product_has_assortment_blacklist_id
- product_has_both_assortment_id

Or something like that.